Pages/homePage.js

```diff
@@ -0,0 +1,50 @@
+var dataJS = require("../data");
+
+class HomePage {
+    constructor() {
+        this.title= "HomePage";
+    }
+    // GETTERS
+    get loginButton (){
+        return browser.driver.findElement(by.className("header-text"));
+    }
+    get loginButtonLocator(){
+        return ".header-text";
+    }
+    get CategoryLocator(){
+        return ".category-item"
+    }
+    get fashionCategory(){
+        return browser.driver.findElement(by.css("div.category-item a"));
+    }
+    get logoutButton (){
+        return browser.driver.findElement(by.css("div.login-acccount button"));
+    }
+    
+    // ACTIONS
+    // This method waits for Category to show
+    waitForCategories(){
+        browser.wait(dataJS.EC.presenceOf($(this.CategoryLocator)), 5000)
+    }
+    // This method waits for link Login to show
+    waitForLoginButton(){
+        return browser.wait(dataJS.EC.presenceOf($(this.loginButtonLocator), 5000))
+    }
+    // This method clicks on link Login
+    clickOnLogin() {
```

Comment:
On some places you have this format "methodName(){  ....." and on other there is this format "methodName() {  ....." 
Please choose one.

Reply:
I fixed this, choosed one format

Reply:
This repeats on other places, please review once again.